ui/components/app/detected-token/detected-token-ignored-popover/detected-token-ignored-popover.js

```diff
@@ -11,13 +11,13 @@ import {
   Text,
   Modal,
   ModalOverlay,
-  ModalContent,
-  ModalHeader,
   Button,
   BUTTON_VARIANT,
   Box,
   BUTTON_SIZES,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 
 const DetectedTokenIgnoredPopover = ({
   partiallyIgnoreDetectedTokens,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 35 53 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/7d74e0a8-c605-4544-a780-a24b947f6054">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 36 07 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/dbcf0702-5687-4bb2-8369-02b6755f84e1">

Reply:
@garrettbear should `HeaderBase` center the text if there is no accessories?